server/db/models/index.js

```diff
@@ -1,18 +1,41 @@
 const User = require('./user')
+const Pothole = require('./pothole')
+const Order = require('./order')
+const Comment = require('./comment')
+const Crew = require('./crew')
+const Upvote = require('./upvote')
+
+Pothole.belongsTo(User, {foreignKey: 'reporterId'})
+
+Order.hasMany(Pothole)
+Pothole.belongsTo(Order)
+
+User.hasMany(Order)
+Order.belongsTo(User)
+
+Crew.hasMany(Order)
+Order.belongsTo(Crew)
+
+Crew.hasMany(User)
+User.belongsTo(Crew)
+
+User.hasMany(Comment)
+Comment.belongsTo(User)
+
+Pothole.hasMany(Comment)
+Comment.belongsTo(Pothole)
+
+Pothole.belongsToMany(User, {through: Upvote})
```

Comment:
Do we need to have Upvote as its own file? Also, I believe that belongsToMany should be called on both so that we can access the magic methods

Reply:
deleted upvote table...created automatically through join table "through" specification (line 28 / 29)